third_party/move/move-compiler/src/parser/ast.rs

```diff
@@ -453,6 +453,17 @@ pub enum NameAccessChain_ {
     // (<Name>|<Num>)::<Name>::<Name>
     Three(Spanned<(LeadingNameAccess, Name)>, Name),
 }
+
+impl NameAccessChain_ {
+    pub fn name(&self) -> &Name {
```

Comment:
Lets call this `base_name` or so to make it clearer. Also add documentation comment for public functions